src/pgduckdb_table_am.cpp

```diff
@@ -398,70 +399,70 @@ duckdb_scan_sample_next_tuple(TableScanDesc scan, SampleScanState *scanstate, Tu
  * ------------------------------------------------------------------------
  */
 
-static const TableAmRoutine duckdb_methods = {.type = T_TableAmRoutine,
-
-                                              .slot_callbacks = duckdb_slot_callbacks,
-
-                                              .scan_begin = duckdb_scan_begin,
-                                              .scan_end = duckdb_scan_end,
-                                              .scan_rescan = duckdb_scan_rescan,
-                                              .scan_getnextslot = duckdb_scan_getnextslot,
-
-                                              /* optional callbacks */
-                                              .scan_set_tidrange = NULL,
-                                              .scan_getnextslot_tidrange = NULL,
-
-                                              /* these are common helper functions */
-                                              .parallelscan_estimate = table_block_parallelscan_estimate,
-                                              .parallelscan_initialize = table_block_parallelscan_initialize,
-                                              .parallelscan_reinitialize = table_block_parallelscan_reinitialize,
-
-                                              .index_fetch_begin = duckdb_index_fetch_begin,
-                                              .index_fetch_reset = duckdb_index_fetch_reset,
-                                              .index_fetch_end = duckdb_index_fetch_end,
-                                              .index_fetch_tuple = duckdb_index_fetch_tuple,
-
-                                              .tuple_fetch_row_version = duckdb_fetch_row_version,
-                                              .tuple_tid_valid = duckdb_tuple_tid_valid,
-                                              .tuple_get_latest_tid = duckdb_get_latest_tid,
-                                              .tuple_satisfies_snapshot = duckdb_tuple_satisfies_snapshot,
-                                              .index_delete_tuples = duckdb_index_delete_tuples,
-
-                                              .tuple_insert = duckdb_tuple_insert,
-                                              .tuple_insert_speculative = duckdb_tuple_insert_speculative,
-                                              .tuple_complete_speculative = duckdb_tuple_complete_speculative,
-                                              .multi_insert = duckdb_multi_insert,
-                                              .tuple_delete = duckdb_tuple_delete,
-                                              .tuple_update = duckdb_tuple_update,
-                                              .tuple_lock = duckdb_tuple_lock,
-                                              .finish_bulk_insert = duckdb_finish_bulk_insert,
+static const TableAmRoutine duckdb_methods = {
+    .type = T_TableAmRoutine,
+
+    .slot_callbacks = WRAP_CPP_FUNC(duckdb_slot_callbacks),
```

Comment:
Why are these needed?

Reply:
Ah I need to revert this, it was an experiment to wrap simply the table am callbacks. But since none of them have CPP code it's not necessary.